readme-vars.yml

```
@@ -32,40 +32,23 @@ param_ports:
   - { external_port: "6881", internal_port: "6881/udp", port_desc: "udp connection port" }
 param_usage_include_env: true
 param_env_vars:
-  - { env_var: "TZ", env_value: "Europe/London", desc: "Specify a timezone to use EG Europe/London"}
-  - { env_var: "WEBUI_PORT", env_value: "8080", desc: "for changing the port of the webui, see below for explanation"}
-
-# optional parameters
-optional_block_1: false
-optional_block_1_items: ""
+  - { env_var: "WEBUI_PORT", env_value: "8080", desc: "for changing the port of the web UI, see below for explanation"}
 
 # application setup block
 app_setup_block_enabled: true
 app_setup_block: |
-  The webui is at `<your-ip>:8080` and the default username/password is `admin/adminadmin`.  
-  
-  Change username/password via the webui in the webui section of settings.  
+  The web UI is at `<your-ip>:8080` and the initial password for the `admin` user will be printed to the container log on startup.
 
-  ### WEBUI_PORT variable
-  
-  Due to issues with CSRF and port mapping, should you require to alter the port for the webui you need to change both sides of the -p 8080 switch AND set the WEBUI_PORT variable to the new port.  
-  
-  For example, to set the port to 8090 you need to set -p 8090:8090 and -e WEBUI_PORT=8090  
-  
-  This should alleviate the "white screen" issue.  
-  
-  If you have no webui , check the file /config/qBittorrent/qBittorrent.conf  
-  
-  edit or add the following lines  
+  You can then change username/password in the web UI section of settings. If you do not change the password a new one will be generated every time the container starts.
```

```suggestion
  You should then change username/password in the web UI section of settings. If you do not change the password a new one will be generated every time the container starts.
```
or even must could work